projects/plugins/jetpack/_inc/client/state/recommendations/reducer.js

```diff
@@ -58,6 +58,20 @@ const data = ( state = {}, action ) => {
 	switch ( action.type ) {
 		case JETPACK_RECOMMENDATIONS_DATA_FETCH_RECEIVE:
 		case JETPACK_RECOMMENDATIONS_DATA_UPDATE:
+			// Filter out deprecated slugs and convert them to new ones (if the new ones don't already exist)
+			for ( const [ key, value ] of Object.entries( action.data ) ) {
+				const actionData = action.data;
+				if ( key === 'site-type-business' ) {
+					actionData[ 'site-type-agency' ] = actionData[ 'site-type-agency' ] || value;
```

Comment:
Now, if `actionData[ 'site-type-agency' ]` is `false`, the variable `value` will be used 🤔 I guess it's not what we want. Besides that the approach looks good!

Reply:
Went ahead and fix that, you can go back and approve this when you have time but I will move to Needs Review since the code freeze is tomorrow